charts/juicefs-s3-gateway/templates/deployment.yaml

```diff
@@ -77,11 +106,19 @@ spec:
                 secretKeyRef:
                   name: juicefs-secret
                   key: secret-key
+            {{- if .Values.envs }}
+            {{ toYaml .Values.envs | nindent 12 }}
```

Comment:
prefer :
```
            {{- if .Values.envs }}
{{ toYaml .Values.envs | trim | indent 12 }}
            {{- end }}
```

`nindent` will introduce a blank line above

Reply:
there're existing `nindent` usages below, not sure which to follow. maybe we can later format all indentations in an another PR

the new line doesn't affect Kubernetes resource definition, lets ignore aesthetics for now?